mypy/test/testtransform.py

```diff
@@ -30,15 +28,19 @@ class TransformSuite(Suite):
                        'semanal-abstractclasses.test',
                        'semanal-python2.test']
 
-    def cases(self) -> List[DataDrivenTestCase]:
+    @classmethod
+    def cases(cls) -> List[DataDrivenTestCase]:
         c = []  # type: List[DataDrivenTestCase]
-        for f in self.transform_files:
+        for f in cls.transform_files:
             c += parse_test_cases(os.path.join(test_data_prefix, f),
                                   test_transform,
```

Comment:
the argument `test_transform`, which corresponds to the parameter  `perform`, is not used in pytests. The parameter is propagated to DataDrivenTestCase which uses it in the method `run()`. This method is not called; maybe we should add `assert False` there.